app/controllers/children_controller.rb

```diff
@@ -36,23 +44,15 @@ def update
   end
 
   def destroy
+    child = current_partner.children.find_by(id: params[:id])
```

Comment:
If there's an accidental double request, this might blow up because child will be nil. Do we want to defend against that?